src/lib/core/CHIPConfig.h

```diff
@@ -1345,6 +1345,18 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_SETUP_CODE_PAIRER_DISCOVERY_TIMEOUT_SECS 30
 #endif // CHIP_CONFIG_SETUP_CODE_PAIRER_DISCOVERY_TIMEOUT_SECS
 
+/**
+ *  @def CHIP_CONFIG_ON_NETWORK_PAIRER_DISCOVERY_TIMEOUT_SECS
+ *
+ *  @brief
+ *    This is the default timeout for the discovery of devices by
+ *    the on network pairer.
```

Comment:
This is too generic. This should be settable via the Python API. It should not be a compile-time constant, and should not be here if it only applies to Python controller.

Reply:
In fact, this problem also occurs on other platforms, but the chip-tool demo limits the default timeout period of 2 minutes for command execution.
But you are right, it should be more reasonable as a parameter